plan/physical_plan_builder.go

```diff
@@ -223,9 +252,7 @@ func (ds *DataSource) findBestTask(prop *requiredProp) (task, error) {
 			}
 		}
 	}
-
-	ds.storeTask(prop, t)
-	return t, nil
+	return
```

Comment:
redundant `return`

Reply:
Delete it will cause error `missing return at end of function`.

Reply:
OK